src/sparseml/pytorch/image_classification/README_image_classification.md

```diff
@@ -0,0 +1,253 @@
+Sparsification is a powerful feature that results in faster, smaller, and cheaper deployable models. 
+The sparse model can be eventually deployed to Neural Magic's [DeepSparse] Engine. 
+This allows running the inference with GPU-class performance directly on your CPU.
+
+This integration enables spinning up one of the following end-to-end functionalities:
+- **Sparsification of Popular Torchvision Models** - easily sparsify any popular [torchvision] image classification models. 
+- **Sparse Transfer Learning** - fine-tune a sparse backbone model (or use one of our [sparse pre-trained models](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=classification&page=1)
+## Installation
+
+We recommend using a [virtualenv] to install dependencies.
+```pip install sparseml[torchvision]```
+
+## Tutorials
+
+- [Sparsifying PyTorch Models Using Recipes](https://github.com/neuralmagic/sparseml/blob/main/integrations/pytorch/tutorials/sparsifying_pytorch_models_using_recipes.md)
+- [Sparse Transfer Learning for Image Classification](https://github.com/neuralmagic/sparseml/blob/main/integrations/pytorch/tutorials/classification_sparse_transfer_learning_tutorial.md)
+
+## Getting Started
+
+### Sparsification of Image Classification Models
+
+[SparseML] Image Classification integration allows sparsifying any dense image classification model.
+
+A dense [ResNet] model is trained on the [ImageNette] dataset in the example below.
+By passing the recipe `zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenette/pruned-conservative?recipe_type=original` 
+(located in [SparseZoo]) we modify (sparsify) the training process and/or the model.
+
+```bash
+sparseml.image_classification.train \
+    --recipe-path "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenette/pruned-conservative?recipe_type=original" \
+    --dataset-path ./data \
+    --pretrained True \
+    --arch-key resnet50 \
+    --dataset imagenette \
+    --train-batch-size 128 \
+    --test-batch-size 256 \
+    --loader-num-workers 8 \
+    --save-dir sparsification_example \
+    --logs-dir sparsification_example \
+    --model-tag resnet50-imagenette-pruned \
+    --save-best-after 8         
+```
+
+### Sparse Transfer Learning
+
+Once you sparsify a model using [SparseML], you can easily sparse fine-tune it on a new dataset.
+While you are free to use your backbone, we encourage you to leverage one of our [sparse pre-trained models](https://sparsezoo.neuralmagic.com) to boost your productivity!
+
+In the example below, we fetch a pruned [ResNet] model, pre-trained on [ImageNet] dataset. We then fine-tune the model on the [ImageNette] dataset. 
+```bash
+sparseml.image_classification.train \
+    --recipe-path zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none?recipe_type=transfer-classification \
+    --checkpoint-path zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none \
```

Comment:
If weights are to be downloaded from SparseZoo then checkpoint-path could be set to `zoo`, here it is needed because we are downloading resnet weights trained on ImageNet and then finetuning on a different dataset, even if we could avoid checkpoint-path, I would still like to keep it explicit here to show the pathway to finetune on a different dataset, does that make sense?